cmd/lotus-chainwatch/processor/reward.go

```diff
@@ -26,15 +31,43 @@ func (p *Processor) setupRewards() error {
 
 	if _, err := tx.Exec(`
 /*
-* captures chain-specific power state for any given stateroot
+* captures base block reward per miner per state root and does not
+* include penalties or gas reward
 */
+create table if not exists base_block_rewards
+(
+	state_root text not null
+		constraint block_rewards_pk
+			primary key,
+	base_block_reward numeric not null
+);
+
+/* captures chain-specific power state for any given stateroot */
 create table if not exists chain_power
 (
 	state_root text not null
 		constraint chain_power_pk
 			primary key,
 	baseline_power text not null
 );
+
+create materialized view if not exists top_miners_by_base_reward as
```

Comment:
Should this be refreshed via the `refreshViews()` method in the processor?

Reply:
This is going to managed with a timer to be executed once per day. I'm working on this pattern now.